test/Destructurama.ByIgnoring.Tests/TestCases/ByIgnoringTestCases.cs

```diff
@@ -0,0 +1,202 @@
+// (C) Copyright 2023 Nullable, Inc. All rights reserved.
+// This file is part of Nullable's product Aware and cannot be copied and/or
+// distributed without the express permission of Nullable, Inc.
```

Comment:
?

Reply:
This was accidentally added. Just pushed a commit that replaces these with the proper copyright notice.